### PR TITLE
chore: 🤖 new routes for storage policy

### PR DESCRIPTION
**Pull request by DhariniJeeva**
✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12081

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12081)

## Description

This PR adds new routes to support storage policies. 
Note: the model hook will be added along with creating the component 

## Screenshots (if appropriate):
**ORG**
Screenshot 2023-12-19 at 1 06 30 PM
Screenshot 2023-12-19 at 1 06 25 PM

**GLOBAL**
Screenshot 2023-12-19 at 1 06 13 PM
Screenshot 2023-12-19 at 1 06 07 PM

## How to Test

:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:

- [x] I have added before and after screenshots for UI changes
- ~[ ] I have added JSON response output for API changes~
- ~[ ] I have added steps to reproduce and test for bug fixes in the description~
- ~[ ] I have commented on my code, particularly in hard-to-understand areas~
- [x] My changes generate no new warnings
- ~[ ] I have added tests that prove my fix is effective or that my feature works~

### CODE REVIEW

File: ui/admin/app/templates/scopes/scope/policies/index.hbs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
```
Comment (calcaide) [inline ui/admin/app/templates/scopes/scope/policies/index.hbs:7]: None blocking comment: Will be awesome if we can add a dummy heder or something similar, so when testing this we can see content that tells us we are on the section.

**Review by calcaide — APPROVED** [review]
Let's gooo!! new section coming soon :smiley:

File: ui/admin/app/templates/scopes/scope/policies/index.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{{!
```
Comment (ZedLi) [inline ui/admin/app/templates/scopes/scope/policies/index.hbs:1]: I think this copyright got added twice

File: ui/admin/app/abilities/scope.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ export default class OverrideScopeAbility extends OverrideModelAbility {
   get canCreate() {
     switch (this.collection) {
       case 'storage-buckets':
+      case 'policies':
```
Comment (ZedLi) [inline ui/admin/app/abilities/scope.js:24]: Should this be called `policies` or `storage-policies`?

**Review by ZedLi — APPROVED** [review]
Looks good!

File: ui/admin/app/abilities/scope.js (hunk 3 above)
Comment (DhariniJeeva) [reply ui/admin/app/abilities/scope.js:24]: the API calls it `policies` so I kept it to be consistent

File: ui/admin/app/templates/scopes/scope/policies/index.hbs (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/app/templates/scopes/scope/policies/index.hbs]: Good point -  added!

**Pull request merged**